### PR TITLE
Improve zFCP related code

### PR DESCRIPTION
**Pull request by jstodola**
Fixes suggested in https://github.com/storaged-project/blivet/pull/1035#pullrequestreview-1023659533

### CODE REVIEW

**Comment by StorageGhoul** [comment]
Can one of the admins verify this patch?

**Comment by jstodola** [comment]
@steffen-maier, can you please review the changes?

File: blivet/zfcp.py [hunk 1]
```diff
@@ -253,9 +251,9 @@ def online_device(self):
         unitdir = "%s/%s" % (portdir, self.fcplun)
         failed = "%s/failed" % (unitdir)
 
-        # Activating an NPIV enabled device using devnum, WWPN and LUN should still be possible
-        # as this method was used as a workaround until the support for NPIV enabled devices has
-        # been implemented. Just log a warning message and continue.
+        # Activating using devnum, WWPN, and LUN despite available zFCP auto LUN scan should still
+        # be possible as this method was used as a workaround until the support for NPIV enabled
+        # devices has been implemented. Just log a warning message and continue.
```
Comment (steffen-maier) [inline blivet/zfcp.py:255]: # be possible as this method was used as a workaround until the support for zfcp auto LUN scan
        # has been implemented. Just log a warning message and continue.

[Sorry, I think I had missed that before.]

**Review by steffen-maier — APPROVED** [review]

**Review by steffen-maier — APPROVED** [review]

**Comment by steffen-maier** [comment]
Oops, sorry for the multiple reviews. I thought I could review each commit separately, but that does not seem to be how github works.

File: blivet/zfcp.py (hunk 1 above)
Comment (jstodola) [reply blivet/zfcp.py]: Fixed.

**Comment by vojtechtrefny** [comment]
Jenkins, ok to test.

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me, thank you.

**Pull request merged**